logic/transaction_pool.js

```diff
@@ -136,8 +142,9 @@ TransactionPool.prototype.transactionInPool = function(id) {
 
 /**
  * Gets an unconfirmed transaction based on transaction id.
- * @param {string} id - Transaction id.
- * @return {(Object|undefined)} - Transaction or undefined.
+ *
+ * @param {string} id - Transaction id
+ * @returns {(Object|undefined)} - Transaction or undefined
```

Comment:
What do the round brackets signify?

Reply:
I don't think they are needed. 
Will be removed with next commits.